test/gui/tst_sharing/test.feature

```diff
@@ -134,3 +134,22 @@ Feature: Sharing
             | path        | /simple-folder |
             | name        | Public link    |
         And the public should not be able to download the file "lorem.txt" from the last created public link by "Alice" on the server
+
+
+    Scenario Outline: change collaborator permissions of a file & folder
+        Given user "Alice" has created on the server folder "simple-folder"
+        And user "Alice" on the server has created file "lorem.txt"
+        And user "Brian" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has shared folder "simple-folder" on the server with user "Brian"
+        And user "Alice" has shared folder "lorem.txt" on the server with user "Brian"
+        And user "Alice" has set up a client with default settings
+        When the user removes permissions "<permissions>" for user "Brian Murphy" of resource "%client_sync_path%/simple-folder" using the client-UI
+        And the user removes permissions "<permissions>" for user "Brian Murphy" of resource "%client_sync_path%/lorem.txt" using the client-UI
+        Then the user should not have "<permissions>" permissions for user "Brian Murphy" for resource "%client_sync_path%/simple-folder"
+        And the user should not have "<permissions>" permissions for user "Brian Murphy" for resource "%client_sync_path%/lorem.txt"
```

Comment:
I think we should check through the API if those permissions really got changed on the server

Reply:
Sorry, my comment was not clear.
I didn't mean to delete those UI test steps, but check additionally with the API
Maybe we can also rephrase those UI steps:
`Then "<permissions>" permissions should not be displayed for user "Brian Murphy" for resource "%client_sync_path%/simple-folder on the UI"`